src/main/scala/io/github/kiemlicz/shelm/Chart.scala

```diff
@@ -41,6 +41,7 @@ object Chart {
     keywords <- c.get[Option[List[String]]]("keywords")
     home <- c.get[Option[URI]]("home")
     sources <- c.get[Option[List[URI]]]("sources")
+    // FIXME: If dependencies are defined in `requirements.yaml` instead of in `Chart.yaml` then they won't be detected
```

Comment:
Helm 2 is not supported, I'm unsure that `requirements.yaml` can be successfully used in Helm 3

Reply:
It seems to work, moreover such usage exists within the repository ([`src/sbt-test/shelm/duplicated/build.sbt`](https://github.com/kiemlicz/shelm/blob/6f894d4708e996ee871ac08a402cb93208aae086/src/sbt-test/shelm/duplicated/build.sbt#L28)) :)
The Keycloak chart does have Postgres as a dependency configured via `requirements.yaml`, and `helm dependency update` indeed tries to fetch it